Updated reponse.js (Conclusion)

satwikgoyal opened the pull request:
Used the QA component and added place holder body and questions field.

satwikgoyal committed:
Updated reponse.js (Conclusion)

Used the QA component and added place holder body and questions field.

rasdy reviewed (changes requested):
Looks good, but we should renamed the whole response page to "conclusion" (file name, component name, title, etc.)

satwikgoyal committed:
Changed everything from response to conclusion

Renamed the whole response page to "conclusion" (file name, component name, title, etc.)

rasdy commented on frontend/src/pages/finalReflection.js:
```diff
@@ -45,17 +45,17 @@ function FinalReflection({ pages, setPages, activePage, setActivePage }) {
-      if (pages.response.completed) {
-        if (!pages.response.visited) {
+  function goToConclusion() {
+    if (pages.conclusion.completed) {
+      if (pages.conclusion.completed) {
```
Double `if`'s?

satwikgoyal replied:
The double ifs were already there but you are right, I should have noticed this. Thanks!

rasdy commented on frontend/src/pages/conclusion.js:
```diff
@@ -0,0 +1,56 @@
+        <Grid item lg={12}>
+          <Box m="2rem">
+          </Box>
+          <QA header={bodyText} questions={questions} pages={pages} nextPageName={"conclusion"} />
```
Hitting `submit` results in an exception

satwikgoyal replied:
Resolved in the latest commit. Thanks a lot for the reviews and sorry about the minor mistakes.

rasdy reviewed (changes requested):
I noticed two small issues

satwikgoyal committed:
Submit button and minor changes

Used the useHistory hook from react-router-dom to redirect the student to the home page after submitting the conclusion response. Made changes in the q&a.js component file to redirect to the home page. In the finalReflection.js there were repeated/double ifs, so removed one of them.

rasdy reviewed (approved):
👍
